extensions/two-factor-authentication-code-generator/src/index.tsx

```diff
@@ -132,6 +136,11 @@ export default function AppsView() {
                 />
               </ActionPanel.Section>
               <ActionPanel.Section>
+                <Action.CopyToClipboard
+                  icon={Icon.Warning}
+                  title="Backup App"
+                  content={JSON.stringify({ name: a.name, secret: a.secret, options: a.options })}
+                />
```

Comment:
I'd prefer to save backups to a file rather than copy them to clipboard. Many apps continuously read the clipboard contents.

This is also a security concern and users should only use this feature when necessary. I think it would be better to create a separate command for this feature and disable it by default.

What do you think?

Reply:
I kinda like that idea 🙂